bin/cli.js

```diff
@@ -13,12 +14,15 @@ const teamActions = require('../src/team.actions');
 const teamsActions = require('../src/teams.actions');
 
 const opn = require('opn');
-const argv = require('yargs')
+const yargs = require('yargs')
   .usage('Usage: jiractl --team=orange-cats [action] [context]')
   .default('output', 'console')
   .alias('t', 'team')
   .alias('o', 'output')
-  .argv;
+  .describe('o', 'Pick output format')
+  .choices('o', ['console', 'json']);
+
+const argv = yargs.argv;
```

Comment:
but you removed argv? does that intrinsically exist off of yargs?

Reply:
Yea, `argv` is always available on `yargs`, but now we need access to both since we want to call the functions on `yargs` and grab the args from `argv`